ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldProcess.cpp

```diff
@@ -292,8 +277,8 @@ void ThermoMechanicalPhaseFieldProcess<
 
     GlobalExecutor::executeSelectedMemberOnDereferenced(
         &ThermoMechanicalPhaseFieldLocalAssemblerInterface::preTimestep,
-        _local_assemblers, pv.getActiveElementIDs(),
-        getDOFTable(process_id), x, t, dt);
+        _local_assemblers, pv.getActiveElementIDs(), getDOFTable(process_id), x,
+        t, dt);
```

Comment:
Just curious, which version of clang(-format) do you use?

Reply:
v6.0.0